Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1993,7 +1993,7 @@ LayoutUnit RenderFlexibleBox::computeCrossSizeForFlexItemUsingContainerCrossSize
         if (minimumSize.isFixed())
             definiteValue = std::max(definiteValue, LayoutUnit { minimumSize.value() });
 
-        return definiteValue;
+        return adjustContentBoxLogicalWidthForBoxSizing(definiteValue, LengthType::Fixed);
```

Comment:
`// Keep this sync'ed with flexItemCrossSizeShouldUseContainerCrossSize()`
does this still apply and if it does, any changes to that function needed now?

Reply:
I think we still need it since it's used in a few different locations right now, but I also added a debug assert to help enforce the comment. AFAICT that function shouldn't need to change since the value of box-sizing shouldn't affect what value it returns